src/entity/src/user.js

```diff
@@ -498,7 +499,13 @@ export default class User {
       client: this.client
     });
 
-    return request.execute()
+    let prm = Promise.resolve();
+
+    if (LiveServiceFacade.isInitialized()) {
+      prm = LiveServiceFacade.unregister();
```

Comment:
I had mistakenly read that you meant to put the check inside `unregister()` of `LiveService`. I like it inside `unregister()` of `LiveServiceFacade`, will put it there.